UPGRADE-3.1.md

```diff
@@ -103,11 +103,10 @@ FrameworkBundle
 
    ```yaml
    framework:
-       serializer: ~
```

Comment:
Wouldn't it be better to keep this? I'm not sure that everyone will think that removing the serializer section will disable it.